tests/unit/model/message_test.py

```diff
@@ -2615,6 +2616,7 @@ def test_copy() -> None:
         message.copy(
             structure=[Link(INITIAL, Field("C")), Link(Field("C"), FINAL)],
             types={Field("C"): RANGE_INTEGER},
+            byte_order={Field("C"): ByteOrder.HIGH_ORDER_FIRST},
```

Comment:
It would make sense to also test a case with `ByteOrder.LOW_ORDER_FIRST`. The current test would be also be fulfilled, if `byte_order` is just ignored by `copy` and the default value is used.

Reply:
done